examples/main.py

```diff
@@ -0,0 +1,98 @@
+#To mimic the functionalities of main.c in ../device-app-boilerplate
+from _my_coco import lib, ffi
+#import device_callback as dc
+import device_declarations as dd
+import numpy as np
+import time
+
+#Parse Cmdline from main.c
+
+app_config = ffi.new("cmdline_params_t *")
```

Comment:
This is not needed. This is used only in case of example app. This is not needed when working with the cocodevicesdk